bHY6yh-d37603/cod-klc89d_bHY6yh-d37603.xml

```diff
@@ -0,0 +1,272 @@
+                 </p>
+              </availability>
+           </publicationStmt>
+           <sourceDesc>
+           <listWit>
+              <witness xml:id="Ka" n="cod-klc89d">Klosterneuberg, Stiftsbibliothek, CCl. 381</witness>        
+           </listWit>
+        </sourceDesc>
+     </fileDesc>
+     <encodingDesc>
+        <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+        <editorialDecl>
+           <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+      guidelines for a diplomatic edition.</p>
+        </editorialDecl>
+     </encodingDesc>
+     <revisionDesc status="draft">
+        <listChange>
+           <change when="2024-11-11" status="draft" n="0.0.0">
+              <p>Created file for the first time.</p>
+           </change>
+        </listChange>
+     </revisionDesc>
+  </teiHeader>
+  <text xml:lang="la">
+     <front>
+        <div xml:id="starts-on">
+           <pb ed="#Ka" n="80-r"/>
+           <cb ed="#Ka" n="a"/>
+        </div>
+     </front>
+      <body>
+         <div xml:id="bHY6yh-d37603"><!-- p2l5-r6 -->
+            <head xml:id="bHY6yh-d37603-Hd1e108">Rubrica 6</head>
+            <head xml:id="bHY6yh-d37603-Hd1e111" type="question-title">De vtilitate oblationis munerum et beneficiorum</head>
+            <p xml:id="bHY6yh-d37603-d1e114">
+              <lb ed="#Ka"/>de utilitate oblationis munerum 
+              <lb ed="#Ka"/>et beneficiorum Rubrica 6
+            </p>
+            <p xml:id="bHY6yh-d37603-d1e121">
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d37603-Rd1e117">MOyses in deuterono capitulo 23</ref> <quote xml:id="bHY6yh-d37603-Qd1e127">fratri 
+              <lb ed="#Ka"/>tuo absque usura id quod indiget 
+              <lb ed="#Ka"/>comodabis ut benedicat tibi dominus 
+              <lb ed="#Ka"/>deus tuus in omni tempore tuo</quote>
+            </p>
+            <p xml:id="bHY6yh-d37603-d1e137">
+              <lb ed="#Ka"/><ref xml:id="bHY6yh-d37603-Rd1e135">Salomon in prouerbiis capitulo 17</ref> <quote xml:id="bHY6yh-d37603-Qd1e141">Munera 
+              <lb ed="#Ka"/>de sinu impius accipit ut conuertat 
+              <lb ed="#Ka"/>semitas iudicii</quote> 
+            </p>
+            <p xml:id="bHY6yh-d37603-d1e150">
+              <ref xml:id="bHY6yh-d37603-Rd1e148">18 capitulo</ref> <quote xml:id="bHY6yh-d37603-Qd1e152">donum hominis 
+              <lb ed="#Ka"/>dilatat viam eius et an principes 
+              <lb ed="#Ka"/>spacium ei facit</quote> //
+            </p>
+            <p xml:id="bHY6yh-d37603-d1e158">
+              <ref xml:id="bHY6yh-d37603-Rd1e158">19 capitulo</ref> // <quote xml:id="bHY6yh-d37603-Qd1e160">amici sunt do<lb ed="#Ka" break="no"/>ne 
```

Comment:
```suggestion
            <p xml:id="bHY6yh-d37603-d1e150">
              <ref xml:id="bHY6yh-d37603-Rd1e148">18 capitulo</ref> <quote xml:id="bHY6yh-d37603-Qd1e152">donum hominis 
              <lb ed="#Ka"/>dilatat viam eius et an principes 
              <lb ed="#Ka"/>spacium ei facit</quote> 
            </p>
            <p xml:id="bHY6yh-d37603-d1e158">
              // <ref xml:id="bHY6yh-d37603-Rd1e158">19 capitulo</ref> // <quote xml:id="bHY6yh-d37603-Qd1e160">amici sunt do<lb ed="#Ka" break="no"/>ne 
```

Reply:
Hi Jeff! I have been doing that going forward as per your previous request. This request was from November. Do you need me to go back and change all of them? Also, there are times, where the "//" is genuinely at the end of an entry and that is very obvious.

Reply:
Ok. When the double slash is at the end of the line and the next entry ends on a new line, then I think you're correct that it is appropriate to keep the double slash at the end of the entry. 

But in many cases the double slash is in the middle of line where a new entry starts. In that case, I prefer the double slash to be at the beginning of the new entry prior to the opening `<ref>`

Reply:
No problem. Do you need me to go back and change all of the requests?

Reply:
I'll merge as is.